src/Utils/util.ts

```diff
@@ -271,7 +271,7 @@ export function hashText(text: string) {
 }
 
 export function isFeatureEnabled(featureString: string | undefined, feature: Const.FeatureStrings) {
-    if (featureString && featureString.toUpperCase().includes(feature)) {
+    if (featureString && featureString.toUpperCase().includes(feature.toUpperCase())) {
```

Comment:
Was assuming the enum definitions would all be defined as upper case (but can't hurt)